dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -263,10 +265,10 @@ static Configuration configureFreemarker() {
    * Returns mapping from the Maven coordinates of BOM members to jar files that are in the
    * dependency tree of the BOM members.
    */
-  private static ImmutableSetMultimap<String, Path> bomMemberToJars(
+  private static ImmutableSetMultimap<String, AnnotatedJar> bomMemberToJars(
```

Comment:
This should be more straight-forward now since the AnnotatedJar already knows its coordinates. It doesn't seem to be.

Reply:
Right.